docs/installation.md

```diff
@@ -64,6 +66,14 @@ which containerd || ( yum-config-manager --add-repo https://download.docker.com/
     # Install containerd if it's not present
 ```
 
+Amazon Linux 2
+
+```bash
+yum install -y e2fsprogs openssh-clients git 
+which containerd || sudo yum install -y docker
```

Comment:
Thanks for the patch -- this line should ideally just install `containerd`, not a full install of `docker`.

There doesn't appear to be a docker or containerd package on a default AmazonLinux 2 system.
*( I only checked with the `amazonlinux:2` docker container's default yum repo setup though )

If you add the docker-ce repo directly from docker, it's possible to install the most up-to-date version of containerd from there, just like the CentOS example:
```suggestion
which containerd || ( yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo && yum install -y containerd.io )
```

Reply:
That was precisely the command I ran before and it did not work. Your suggested changes dont work. Refer to https://github.com/weaveworks/ignite/issues/670.

Reply:
Ah thanks! Sorry about that -- here's a better fix:
```suggestion
amazon-linux-extras enable docker
yum install containerd
```

I was able to verify this works within an AL2 docker container

Reply:
So yes that works. And that would work fine with docker hub stuff. But when I try it with ECR I cant get it to work : https://github.com/weaveworks/ignite/issues/673

Reply:
Anyways I still applied it.

Reply:
Thanks for making the install uniform for containerd.
Also appreciate the other bug you opened about registry auth.

I'm glad the docker runtime is helping you get around that :+1:

Reply:
Yea so about that, I think we need some instructions for runtime modes and more commands like ignite pull. I will be happy to write that down, but where should I put it?